pymultimatic/model/timeprogram.py

```diff
@@ -74,6 +94,34 @@ class TimeProgramDay:
 
     settings = attr.ib(type=List[TimePeriodSetting])
 
+    def complete_empty_periods(self, setting_mode_for_completion: SettingMode) -> None:
```

Comment:
I have some trouble to understand that.

Does it means you have a "gap" in the timeprogram ?
What is the target temperature during the gap ? 
Isn't it possible to guess the target temperature( since we know the setting mode) ? 

I think it might cause trouble into HA if there is not target temperature

Reply:
Yes, it is for HA that I made this completion so that I have a similar program. For example, see the file test/files/responses/zones_senso.json.
The reduced temperature is 19 "heating/configuration/setback_temperature_setpoint" and the temperature programs are only defined, on weekdays, from 6:00 to 21:00 for a temperature of 21. The holes from 00:00 to 06:00 and 21:00 to 24:00 (time expected by the API) with a temperature of 19 are therefore completed.